target/sim/sim.mk

```diff
@@ -52,6 +55,9 @@ else
 	RUN_AND_EXIT := run -all; exit
 endif
 
+# Carfield config target.
+common_targs += -t $(CARFIELD_CONFIG)
```

Comment:
Why not in bender-common.mk

Reply:
Because the CARFIELD_CONFIG can be passed through the command line in order keep the same flexibility you have for the FPGA.

Reply:
This was resolved.